mcux/CMakeLists.txt

```diff
@@ -56,12 +62,12 @@ if (${MCUX_DEVICE} MATCHES "LPC|MIMXRT6")
   endif()
 endif()
 
-if (${MCUX_DEVICE} MATCHES "MIMXRT117")
+if (${MCUX_DEVICE} MATCHES "MIMXRT11")
```

Comment:
What case is this trying to cover?

Reply:
The soc file for the RT11xx series (soc/arm/nxp_imx/rt/soc_rt11xx.c) makes use the the functions defined in these HAL drivers, whereas the soc file for the RT10xx series does not. This case pulls in those files for RT11xx devices to support the soc file.

As for why the SOC file itself calls those functions, it is based on the board initialization code within our MCUXpresso SDK.